conf/modules/tlsf.xml

```diff
@@ -17,6 +17,6 @@
     <define name="MAX_FLI" value="$(TLSF_MAX_FLI)"/>
     <file_arch name="tlsf_malloc_arch.c"/>
     <include name="$(PAPARAZZI_SRC)/sw/ext/tlsf"/>
-    <file name="tlsf.c" dir="$(PAPARAZZI_SRC)/sw/ext/tlsf"/>
+    <file name="tlsf.c" dir="../../tlsf"/>
```

Comment:
This doesn't look quite right. As far as I understand it this does "work" but maybe keep the old line commented here for future people (just in case this causes issues in the future).

Reply:
this was working for sure, can it be related to the `vpath` tool (again) ?

Reply:
it works, it just looks really weird, I would have guessed it would be something like ../ext/tlsf but that didn't work. I am fine with it like this though. Indeed it needs to be changed due to the vpath tool.

Reply:
I have tried again with the original line, and it does continue compiling, but it gives the following error in the beginning:

../../sw/tools/find_vpaths.py arm-none-eabi-gcc /home/ewoud/paparazzi_check/var/aircrafts/CYFOAM/ap_tmp.list /home/ewoud/paparazzi_check
arm-none-eabi-gcc: error: ERROR:: No such file or directory
arm-none-eabi-gcc: error: found: No such file or directory
arm-none-eabi-gcc: error: more: No such file or directory
arm-none-eabi-gcc: error: than: No such file or directory
arm-none-eabi-gcc: error: once:: No such file or directory
arm-none-eabi-gcc: error: /home/ewoud/paparazzi_check/sw/ext/tlsf/tlsf.c/home/ewoud/paparazzi_check/sw/airborne/arch/chibios/mcu_arch.c: Not a directory

This if statement triggers 34 times for the first srcs[i]:
(line 58)             if os.path.isfile(path.join(vpath, f)) :

I don't really understand what the vpath file does...

Reply:
Ah this looks like #2182 !
Running it from the command line helped to look at the prints and errors:
python3 sw/tools/find_vpaths.py arm-none-eabi-gcc /home/ewoud/paparazzi_check/var/aircrafts/CYFOAM/ap_tmp.list

Reply:
`find_vpaths` again. Does it get better with the latest fix in master? (just rebase master on top this PR)

Reply:
It does get better! Thanks @podhrmic !